style.qss

```diff
@@ -0,0 +1,17 @@
+/*
+Copyright (c) 2016 Shotgun Software Inc.
+ 
+CONFIDENTIAL AND PROPRIETARY
+ 
+This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit 
+Source Code License included in this distribution package. See LICENSE.
+By accessing, using, copying or modifying this work you indicate your 
+agreement to the Shotgun Pipeline Toolkit Source Code License. All rights 
+not expressly granted therein are reserved by Shotgun Software Inc.
+*/
+
+QLabel {
+    font-family: "Open Sans";
```

Comment:
is this part of the 3dsmax distro? Would be great with some comments explaining why we are expicitly setting the fonts for 3dsmax (it's not done in other DCCs).